cloudbuild.yaml

```diff
@@ -6,7 +6,7 @@ steps:
       - "-e"
       - "-c"
       - |
-        git clone -b $BRANCH_NAME https://github.com/getsentry/self-hosted.git
+        git clone -b $BRANCH_NAME $_HEAD_REPO_URL
```

Comment:
Hm, is this documented as supported? I am concerned about the starting underscore 😅

Reply:
It has an underscore since it's only for pull request triggers, but our cloud build only runs on that so it should be fine

Reply:
Ok great!